nuxeo-core/nuxeo-core-api/src/test/java/org/nuxeo/ecm/core/api/security/TestACL.java

```diff
@@ -46,6 +46,12 @@ public void testGetName() {
         assertEquals("test acl", acl.getName());
     }
 
+    @Test
+    public void testSetNullName() {
+        acl = new ACLImpl();
+        assertEquals(ACL.LOCAL_ACL, acl.getName());
+    }
```

Comment:
You're not giving a null name by doing that as the default constructor gives `LOCAL_ACL` as name.

Furthermore, it can make sense to capture the logs and check that the debug log `ACL name is null` is written.